docs/source/howto/installation_guide.rst

```diff
@@ -51,7 +51,14 @@ which will install pyAFQ locally in editible mode and include extensions.
 .. note::
 
   It is also recommended that you utilize python virtual environment and
-  package mangagement tools and begin with a clean environment.
+  package mangagement tools (e.g., conda) and begin with a clean environment.
+
+.. note::
+
+  You should also install the following optional packages before working through
+  the examples: jupyter, fury, AFQ-browser, matplotlib, cloudknot, "pyAFQ[plot]" (with the quotation marks).
+
+  Docker should also be running in the background for the examples involving cloudknot.
```

Comment:
I think that another way to say this would be: 

```suggestion
  Some of the examples in the documentation require additional dependencies. To install these, you can run `pip 
  install pyAFQ[plot]`, which will include visualization tools that are required in these examples. For examples 
  involving the cloudknot distributed computing library, you will also need to set up an [AWS account]([Create Account - aws.amazon.com](https://aws.amazon.com/resources/create-account/)) and have [docker](https://www.docker.com/) installed.
```